src/components/Dropdown/props.ts

```diff
@@ -230,5 +231,15 @@ export type DropdownProps = {
     * The default behavior is to create a div element and append it at the end of the body,
     *  but you can reset it to the scrolling area and make a relative reposition.
     */
-   getPopupContainer?: (triggerNode: HTMLElement) => HTMLElement
+   getPopupContainer?: (triggerNode: HTMLElement) => HTMLElement,
+
+  /**
+    * The placement of the dropdown menu, one of:
```

Comment:
there is one too many space here

```suggestion
   * The placement of the dropdown menu, one of:
```